packages/core/src/registries/designTokenRegistry.ts

```diff
@@ -32,3 +42,11 @@ const resolveSimpleDesignToken = (templateString: string) => {
   const resolvedValue = spacingValues[designKeys[1]] as string;
   return resolvedValue || '0px';
 };
+
+const resolveBorderDesignToken = (templateString: string) => {
+  const nonTemplateValue = templateString.replace(templateStringRegex, '$1');
+  const designKeys = nonTemplateValue.split('.');
+  const designValues = designTokensRegistry[designKeys[0]] as DesignTokensDefinition;
+  const resolvedValue = designValues[designKeys[1]] as string;
```

Comment:
I think `as string` should be changed to match the resolved value being the borders object from DesignTokenDefinition.

```suggestion
  const resolvedValue = designValues[designKeys[1]] as DesignTokensDefinition['borders'];
```